homeassistant/components/zwave.py

```diff
@@ -417,11 +419,11 @@ def test_network(service):
         """Test the network by sending commands to all the nodes."""
         NETWORK.test()
 
-    def stop_zwave(event):
+    def stop_zwave(service):
```

Comment:
You can't just call it `service`, it will still also be called by the event listener, in case which the parameter is an event. Just call it `_` (indicating that it will not be used).